VivianDoreen/tic-tac-toe
# #168868396 Refactor code to improve on the quality

**What does this PR do**
This PR implements refactors the the tic tac toe game

**Description of the task to be done **
Refactor code to improve on the quality
**How should this be manually tested?**
- Clone  the repo with the command:- git clone https://github.com/VivianDoreen/tic-tac-toe.git
- Checkout to the branch with command:- git checkout   ft-candidate-play-against-computer-168826849
- Check the files changed

**What are the relevant pivotal tracker stories**
[#168868396](https://www.pivotaltracker.com/story/show/168868396)

**Any backgound context you may want to provide**
N/A

**Snapshots (if any available)**
N/A
